base/setup/usetup/lang/es-ES.h

```diff
@@ -114,27 +114,28 @@ static MUI_ENTRY esESWelcomePageEntries[] =
     {
         6,
         11,
-        "Este paso de la instalaci\242n copiar\240 ReactOS en el equipo y",
+        "Este es el primer paso de instalaci\242n, que copiar\240 ReactOS en",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         6,
         12,
-        "prepara el segundo paso de la instalaci\242n.",
+        "el equipo y preparar\303\241 el segundo paso de instalaci\242n.",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         15,
-        "\x07  Press ENTER to install or upgrade ReactOS.",
+        "\x07  Pulse INTRO para instalar o actualizar ReactOS.",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         17,
+//      "\x07  Pulse R para reparar una instalación de ReactOS usando la Consola de Recuperación.",
```

Comment:
The menu entry we currently have does not refer to the recovery console, so the existing one is probably fine.

Reply:
The en_US.h includes... I toke that version as base. Someone erase it in es_ES.h, so... Whatever you want.

Reply:
In the en_US.h it's in the line 133.

Reply:
Ah okay. It's fine to keep it then I suppose.

Reply:
I don't know, maybe it's something that could be implemented in the future and they keep the text, I guess?
Happened the same in the 1029th line in es_ES.h and in the 1024th line in en_US.h